packages/cli/src/lib/Cli.ts

```diff
@@ -57,6 +59,10 @@ export class Cli {
     switch (command) {
       case 'help':
         return await help(parsedArgs);
+      case 'doctor':
+        return await doctor();
+      case 'help':
+        return await updateConfig(parsedArgs);
```

Comment:
Wait, what is included in this PR? There seem to be a lot of changes?

Reply:
It should be 'updateConfig' instead of 'help'.
I added this in my last PR (added doctor and updateConfig commands), but for some reason it isn't in the code so I added it again

Reply:
Does this PR replace #220? Should we close the other one?

Reply:
It includes it, but #220 was after I finished writing Doctor and Update and this PR added the Mock Log and Prompt